packages/bitfield/tsup.config.ts

```diff
@@ -0,0 +1,3 @@
+import { createTsupConfig } from '../../scripts/tsup.config';
+
+export default createTsupConfig({ globalName: 'SapphireBitField', target: 'es2020' });
```

Comment:
```suggestion
export default createTsupConfig({ globalName: 'SapphireBitField' });
```

We can target ES2021